client/src/store/store.test.ts

```diff
@@ -0,0 +1,64 @@
+import { reducer, Team, Store, ActionType } from "./store";
+
+describe("reducer", () => {
+  let mockStore: Store;
+  beforeEach(() => {
+    mockStore = { teams: {}, user: { favoriteTeam: null } };
+  });
+
+  test("Bad Action", () => {
```

Comment:
The component tests were really really good. This test seems to lack a bit of description. It says what is being tested, but not what is supposed to happen. I can see that a Bad Action throws an exception, but I'd rather not have to parse the test (unless I'm actively trying to figure out why the reducer is throwing an exception).

Not a blocker . . . just a request for the future.